### DISCUSSION TITLE
How to disable/dim LED for ROCCAT Kone Pure Ultra?

### DISCUSSION THREAD

**Discussion by Cybugs** [X3n0m0rph59/eruption#74] [Q&A] [answered]
Hi,

does someone know how to disable or dim the LED of this mouse? You really need sunglasses if you take your hand of it...

Thanks in advance!

**Comment by X3n0m0rph59** [comment, marked as answer]
Hi @Cybugs, thanks for posting this question!

It currently is only possible to dim the LEDs globally, either by using the CLI or the by using the slider at the top of the window of the Eruption GUI.

I have added an issue (#75) to track the progress of this feature request.

**Reply by Cybugs** [reply]
Thanks a lot. Just tried the new feature and dim-zone.lua works great. With opacity=0.95 the LED on my Kone Ultra is finally eye-friendly.
And also the eruption-gui is now working for me (Manjaro 21.1.0).
